### PR TITLE
docs: import issues template from `ferdium-recipes`

### PR DESCRIPTION
**Pull request by nathanaelhoun**
#### Pre-flight Checklist

Please ensure you've completed all of the following.

- [x] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-app/blob/develop/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/develop/CODE_OF_CONDUCT.md) that this project adheres to.

#### Motivation and Context

As discussed in the maintainer team, we are merging the recipes issues tracker into the app issue tracker, to reduce duplicates and simplify management.

This PR contains the updated issues templates, with some tweaks to remove dead links and redondant informations.

The result can be seen (and tested) in this fork: https://github.com/nathanaelhoun/ferdium-app/issues/new/choose

#### Migration checklist

- [x] Add a recipe tag in ferdium-app (in combinaison with bug for recipes bugs and enhancement for recipes requests). 
- [x] Migrate the issues template from ferdium-recipes to ferdium-app
- [x] Add note in ferdium-recipes to redirect to the main issue tracker (https://github.com/ferdium/ferdium-recipes/pull/320)
- [x] Remove the mentions of ferdium-recipes in ferdium-app issues template
- [x] Rename this link in the code https://github.com/ferdium/ferdium-app/blob/develop/src/config.ts#L330
- [x] Mirror issue labels on the ferdium-recipes to match ferdium-apps' to keep them applied
- [x] Use the [github-cli](https://docs.github.com/en/issues/tracking-your-work-with-issues/transferring-an-issue-to-another-repository?tool=cli) tool to migrate all opened issues from ferdium-recipes to ferdium-app
- [x] Update and pin this issue in ferdium-recipes: https://github.com/ferdium/ferdium-recipes/issues/321
- [x] ~~Disable issues in the recipes repo~~ useless because the new issue templates for recipes prevent to create an issue, and so we can keep the history of closed issues.

#### Release Notes

Merged issues tracker for the main app and recipes to reduce duplicates

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/something_else_open_issue.yml [hunk 1]
```diff
@@ -1,13 +1,12 @@
-name: Open empty issue
+name: Something else? Open empty an issue
```
Comment (vraravam) [inline .github/ISSUE_TEMPLATE/something_else_open_issue.yml]: "Open an empty issue"

**Review by vraravam — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/something_else_open_issue.yml (hunk 1 above)
Comment (nathanaelhoun) [reply .github/ISSUE_TEMPLATE/something_else_open_issue.yml]:
```suggestion
name: Open empty an issue
```

I renamed it to be the last one in the list, but in fact the order is based on the filename, so it's OK

**Review by vraravam — APPROVED** [review]

**Review by vraravam — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/something_else_open_issue.yml (hunk 1 above)
Comment (nathanaelhoun) [reply .github/ISSUE_TEMPLATE/something_else_open_issue.yml]: Oups my bad I misread your correction

**Comment by nathanaelhoun** [comment]
All opened issues of `ferdium-recipes` has been migrated to `ferdium-app` successfully!

**Comment by mh166** [comment]
> ~Disable issues in the recipes repo~ useless because the new issue templates for recipes prevent to create an issue, and so we can keep the history of closed issues.

I'm not so sure about that... 😉

When opening [.../ferdium-app/issues/new](https://github.com/ferdium/ferdium-app/issues/new), it redirects to the template selection page. When opening [.../ferdium-recipes/issues/new](https://github.com/ferdium/ferdium-recipes/issues/new), it stays on that page and you're free to submit a new issue, despite the template saying otherwise. Have you configured anything differently in `ferdium-recipes` than in `ferdium-app`?

**Comment by Alphrag** [comment]
> When opening [.../ferdium-app/issues/new](https://github.com/ferdium/ferdium-app/issues/new), it redirects to the template selection page. When opening [.../ferdium-recipes/issues/new](https://github.com/ferdium/ferdium-recipes/issues/new), it stays on that page and you're free to submit a new issue, despite the template saying otherwise. 

For me using these url both directs to an issue creation without any template. If you however go to `[repo]/issues/new/choose` then you will see the possible templates appear, and in the `ferdium-recipes` repository, this will redirect you to the `ferdium-app` issues.  It seems that github works that way, and I can't see a way to remove/redirect the internal url `[repo]/issues/new` unfortunately. However, we expect people to open issues by going on github and clicking on "New issue" which will trigger the templates. 
How did you get to these urls directly? If this is encoded somewhere, please let us know and we will change them for the appropriate placeholders, so that it works as expected 🙂

**Comment by mh166** [comment]
Sorry for the late reply. Found two ways to get there, one of which you could fix. ;)

The easy one would be an update to [ferdium-recipes/docs/integration.md on L9](https://github.com/ferdium/ferdium-recipes/blob/28e1513d4fc8f265796c115f62b5553bcf2e8079/docs/integration.md?plain=1#L9). I thought about just opening a PR, but I wasn't sure if any of the docs would need remodeling anyway, because of the shift in handling stuff. 🤷

The other one, which originally led me astray, comes down to faulty behavior on GitHub's side. I reported it to them already:
1. Make sure you're not logged in (e.g. use incognito tab)
2. Open a repo with issue templates enabled, e.g. https://github.com/ferdium/ferdium-recipes/issues
3. Click on the 'New issue' link
4. In the new popup, choose to Sign in
5. Already now, the `return_to` parameter shows the wrong URL: `%2Fferdium%2Fferdium-recipes%2Fissues%2Fnew` (i.e., `%2Fchoose` is missing at the end)
6. After the login, you'll find yourself on the wrong page

HTH

**Comment by nathanaelhoun** [comment]
Thanks for the details report @mh166. The link issue in `integration.md` has been fixed, and for the other, we'll wait on GitHub to fix this behaviour. Thanks for reporting it!

**Pull request merged**